IBPSA/Fluid/Chillers/ModularReversible/RefrigerantCycle/TableData2D.mo

```diff
@@ -0,0 +1,209 @@
+    connect(reaPasThrTConIn.y, tabQUse_flow.u1)
+      annotation (Line(points={{50,79},{50,70},{56,70},{56,62}}, color={0,0,127}));
+    connect(reaPasThrTConIn.y, tabPEle.u1)
+      annotation (Line(points={{50,79},{50,70},{96,70},{96,62}}, color={0,0,127}));
+    connect(reaPasThrTEvaOut.y, tabPEle.u2) annotation (Line(points={{-10,79},{
+            -10,70},{84,70},{84,62}},         color={0,0,127}));
+    connect(reaPasThrTEvaOut.y, tabQUse_flow.u2) annotation (Line(points={{-10,79},
+            {-10,70},{44,70},{44,62}},                color={0,0,127}));
+    connect(reaPasThrTEvaIn.y, tabQUse_flow.u2) annotation (Line(points={{-50,79},
+            {-50,70},{44,70},{44,62}},            color={0,0,127}));
+    connect(reaPasThrTEvaIn.y, tabPEle.u2) annotation (Line(points={{-50,79},{-50,
+            70},{84,70},{84,62}},             color={0,0,127}));
+  end if;
+  annotation (Icon(graphics={
+    Line(points={{-60.0,40.0},{-60.0,-40.0},{60.0,-40.0},{60.0,40.0},
+      {30.0,40.0},{30.0,-40.0},{-30.0,-40.0},{-30.0,40.0},{-60.0,40.0},
+      {-60.0,20.0},{60.0,20.0},{60.0,0.0},{-60.0,0.0},{-60.0,-20.0},
+      {60.0,-20.0},{60.0,-40.0},{-60.0,-40.0},{-60.0,40.0},{60.0,40.0},
+      {60.0,-40.0}}),
+    Line(points={{0.0,40.0},{0.0,-40.0}}),
+    Rectangle(fillColor={255,215,136},
+      fillPattern=FillPattern.Solid,
+      extent={{-60.0,20.0},{-30.0,40.0}}),
+    Rectangle(fillColor={255,215,136},
+      fillPattern=FillPattern.Solid,
+      extent={{-60.0,0.0},{-30.0,20.0}}),
+    Rectangle(fillColor={255,215,136},
+      fillPattern=FillPattern.Solid,
+      extent={{-60.0,-20.0},{-30.0,0.0}}),
+    Rectangle(fillColor={255,215,136},
+      fillPattern=FillPattern.Solid,
+      extent={{-60.0,-40.0},{-30.0,-20.0}})}), Documentation(revisions="<html><ul>
+  <li>
+    <i>October 2, 2022</i> by Fabian Wuellhorst:<br/>
+    Adjusted based on the discussion in this issue <a href=
+    \"https://github.com/ibpsa/modelica-ibpsa/issues/1576\">#1576</a>)
+  </li>
+  <li>
+    <i>May 22, 2019,</i> by Julian Matthes:<br/>
+    First implementation (see issue <a href=
+    \"https://github.com/RWTH-EBC/AixLib/issues/715\">AixLib #715</a>)
+  </li>
+</ul>
+</html>", info="<html>
+<p>
+  This model uses two-dimensional table data typically given
+  by manufacturers as required by e.g. European Norm 14511
+  or ASHRAE 205 to calculate
+  <code>QEva_flow</code> and <code>PEle</code>.
+</p>
+<p>
+  For different condenser outlet and evaporator inlet temperatures,
+  the tables must provide two of the three following values:
+  electrical power consumption, evaporator heat flow rate, and COP.
+</p>
+<p>
+  Note that losses are often implicitly included in measured data.
+  In this case, the frosting modules should be disabled.
+</p>
+
+<h4>Scaling factor</h4>
+<p>
+For the scaling factor, the table data for evaporator heat flow rate
+is evaluated at nominal conditions. Hence, the scaling factor is</p>
+<pre>
```

Comment:
@FWuellhorst : I added this new documentation based on today's feedback. Can you please check if I understood and explained it correctly.

Reply:
Yes. I change the `QUse_flow_nominal` to e.g. `QConTabDat_flow` to indicate the call to the table data. Is this fine for you?

Reply:
Yes, please make this change.

Reply:
I did already 👍